systers_portal/templates/community/page.html

```diff
@@ -0,0 +1,15 @@
+{% extends "community/base.html" %}
+{% block title %}
+- {{ community }}
```

Comment:
We want the title to contain both the community name and the community page name.